src/language-js/print/call-arguments.js

```diff
@@ -117,12 +117,10 @@ function printCallArguments(path, options, print) {
         iterateCallArgumentsPath(path, (argPath, i) => {
           if (shouldGroupFirst && i === 0) {
             printedExpanded = [
-              [
-                print([], { expandFirstArg: true }),
-                printedArguments.length > 1 ? "," : "",
-                hasEmptyLineFollowingFirstArg ? hardline : line,
-                hasEmptyLineFollowingFirstArg ? hardline : "",
-              ],
```

Comment:
Every element of the `printedExpanded` array corresponds to an argument of the call. So, for example, `printedExpanded[0]` is the first argument. `printedExpanded[0]` is used in the code below and your change changes the meaning of that code.

Reply:
Oh..., sorry can you show me an example of how it would actually change the meaning of the code?

Reply:
I'll try, but this doesn't look easy

Reply:
Okay, thanks

Reply:
**Prettier 2.3.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAjAlgMwgJygYR3SgHM4cBZCAZwAs4ATABXIAcAbOADwGUBDAO4NyuAkVLkqdRixwdu-IeQCKA8gAp1ASgAEAXgB8O4AF8ANDoBMABmtaQZkBFYx00aslB8cOCAKbeCB4ofOwCfACeHo6oOHxgANZwMDys8cQkyDA4AK5wjnAAtqiMwgwAMnykOXxkAGK4hXwwrqTIIHw5MBAOILQwhewA6rTo8NRpYHA8QWPoAG5jEe1g1NEgxNTkMExxJE3ImKFbjgBW1FwAQnGJyfyFcOXEcIfH+SDnvBmcyjkQ8K92CcQGkcFscO1UHwSuxeqxxDAhugGDBaMgABzWRzwmhwIZxVjteFwcHzF6OACOf3gu2cwQ61AAtFA4IxGL0cHAqehObtagckEcge8toV0FlciLvnBfv8XoK3o4YNCkSi0UhLEq4uh2Bl8BBCgKQCSAKy9HJbAAq0OCQuB8zyAEkoMJYDwwEQXABBF08GARTiArYmExAA)
<!-- prettier-ignore -->
```sh
--parser babel
```

**Input:**
<!-- prettier-ignore -->
```jsx
bifornCringerMoshedPerplexSawderornCringerMoshedPerplexSawderQwer(() => {}, 200)
```

**Output:**
<!-- prettier-ignore -->
```jsx
bifornCringerMoshedPerplexSawderornCringerMoshedPerplexSawderQwer(() => {},
200);

```
vs
**Prettier pr-10941**
[Playground link](https://deploy-preview-10941--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAjAlgMwgJygYR3SgHM4cBZCAZwAs4ATABXIAcAbOADwGUBDAO4NyuAkVLkqdRixwdu-IeQCKA8gAp1ASgAEAXgB8O4AF8ANDoBMABmtaQZkBFYx00aslB8cOCAKbeCB4ofOwCfACeHo6oOHxgANZwMDys8cQkyDA4AK5wjnAAtqiMwgwAMnykOXxkAGK4hXwwrqTIIHw5MBAOILQwhewA6rTo8NRpYHA8QWPoAG5jEe1g1NEgxNTkMExxJE3ImKFbjgBW1FwAQnGJyfyFcOXEcIfH+SDnvBmcyjkQ8K92CcQGkcFscO1UHwSuxeqxxDAhugGDBaMgABzWRzwmhwIZxVjteFwcHzF6OACOf3gu2cwQ61AAtFA4IxGL0cHAqehObtagckEcge8toV0FlciLvnBfv8XoK3o4YNCkSi0UhLEq4uh2Bl8BBCgKQCSAKy9HJbAAq0OCQuB8zyAEkoMJYDwwEQXABBF08GARTiArYmExAA)
<!-- prettier-ignore -->
```sh
--parser babel
```

**Input:**
<!-- prettier-ignore -->
```jsx
bifornCringerMoshedPerplexSawderornCringerMoshedPerplexSawderQwer(() => {}, 200)
```

**Output:**
<!-- prettier-ignore -->
```jsx
bifornCringerMoshedPerplexSawderornCringerMoshedPerplexSawderQwer(
  () => {},
  200
);

```

Reply:
I didn't notice this sorry. I got it. Thanks.